src/pages/organism/navBar/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```

Comment:
This should not be an organism because it does have any state of its own which its changing, 
A molecule can have multiple molecules.